bin/run-marina.R

```diff
@@ -83,7 +83,7 @@ num_permutations <- 1000
 # 
 signature = get.signature(expset.obj, regul, opt$test_phenotype, opt$reference_phenotype, regul.minsize=regulon_minsize, num.permutations=num_permutations)
 save.image(file=paste(opt$output, "/", "master-reg.RData", sep=""))
-q()
+#q() -- Alana commented this out to run the full analysis
```

Comment:
I'm not sure why there was a q() here originally-- is there some useful abbreviated analysis that can be run, or was it just left over from debugging, or something?